x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -37,6 +37,10 @@
 import java.util.stream.Collectors;
 
 public class ReservedRolesStore implements BiConsumer<Set<String>, ActionListener<RoleRetrievalResult>> {
+    public static final String LEGACY_ALERTS_INDEX = ".siem-signals*";
+    public static final String LEGACY_ALERTS_INDEX_CCS = "*:.siem-signals*";
```

Comment:
I had trouble determining if the additional CCS index pattern is necessary considering `all` privileges for `.siem-signals` and `.alerts` grants CCS privileges. Looking for feedback here from people in-the-know.